lib/rubocop/cop/lint/number_conversion.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop warns the usage of unsafe number conversions. Unsafe
+      # number conversion can cause unexpected error if auto type conversion
+      # fails. Cop prefer parsing with number class instead.
+      #
+      # @example
+      #
+      #   # bad
+      #   foo.to_i
+      #   If foo is not parsable then #to_i would return 0 and silently
+      #   ignore the error.
+      #   'alpha_number1c'.to_i => 0
+      #
+      #   "10".to_i
+      #   "10.2".to_f
+      #   "10".to_c
+      #
+      #   # good
+      #   Integer(foo)
+      #   If foo is not parsable then it would throw `ArgumentError` error.
+      #   'alpha_number1c'.to_i => invalid value for Integer(): "hello"
+      #
+      #   Integer("10)
```

Comment:
Double quote is missing.

```diff
-Integer("10)
+Integer("10")
```

Reply:
Fixed it, thanks, I was stuck for some time 🤖